tests/run_pr_tests.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash -xe
+ulimit -n 2048 || true
```

Comment:
why we need this?

Reply:
There seems to be a file descriptor leak in `httpretty` library that tests use to replay HTTP requests/responses. The library looks unmaintained, so PyGithub 2.x replaced it with `response`. I have a local branch with latest PyGithub,  waiting for https://github.com/PyGithub/PyGithub/pull/3377 to be merged (this will fix pagination in a few places).